phpBB/phpbb/install/helper/navigation/navigation_provider.php

```diff
@@ -0,0 +1,115 @@
+<?php
```

Comment:
in some cases it can hurt performances very hard (when a performance is passed instead of a raw variable or the other side) and it often (always?) make the code more complicated to understand.